transaction/examples/access_rule/access_rule.rtm

```diff
@@ -5,22 +5,7 @@ SET_OWNER_ROLE
 LOCK_OWNER_ROLE
```

Comment:
We still have lock owner role?

Reply:
Yup, as owner role is a little special and should be able to go from `OwnerRole::Updatable` to `OwnerRole::Fixed`.